### ISSUE TITLE
homebrew tap for ocm-cli

### ISSUE THREAD

**Issue by cblecker** [openshift-online/ocm-cli#63]
We should create an official tap for ocm-cli, rather than running off my personal tap

This can be facilitated with goreleaser: https://goreleaser.com/customization/#Homebrew

I can help with this task if necessary :)

**Comment by jhernand** [comment]
@cblecker would it make sense to have a new `homebrew-tap` project in the `openshift-online` organization to hold the formula for `ocm` (and maybe for other related tools in the future)?

**Comment by cblecker** [comment]
Yup! That would be good (and similar to my setup here: https://github.com/cblecker/homebrew-tap)

Then folks would `brew tap openshift-online/tap` and `brew install openshift-online/ocm`

**Comment by jhernand** [comment]
OK. @kbsingh can you create that `homebrew-tap` project and make @cblecker and me collaborators?
